### DISCUSSION TITLE
`mdformat-myst` maintainers?

### DISCUSSION THREAD

**Discussion by mattkang** [executablebooks/meta#1208] [General] [answered]
@agoose77 Who's maintaining [mdformat-myst](https://github.com/executablebooks/mdformat-myst/) now?

Have [not been able](https://github.com/executablebooks/mdformat-myst/pull/28#issuecomment-2278255148) to reach @jpmckinney @hukkin @hukkinj1 @chrisjsewell

**Comment by jpmckinney** [comment]
I am not a maintainer to begin with.

**Comment by KyleKing** [comment, marked as answer]
I've been added as a maintainer!

https://github.com/orgs/executablebooks/discussions/1196
